language/signatures.rst

```diff
@@ -335,7 +335,7 @@ As an aid, use the following table:
 | one         | some        | Each B used exactly once, each A used at |
 |             |             | least once                               |
 +-------------+-------------+------------------------------------------+
-| one         | one         | Only satisfiable if #A = #B, bijection   |
+| one         | one         | Only satisfable if #A = #B, bijection   |
```

Comment:
This was was originally correct and is now a typo.

Reply:
Right, fixed.